intellij/workspace/update/.intellij/config/inspection/Default.xml

```diff
@@ -1,5 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <profile version="1.0">
-  <option name="myName" value="Default" />
-  <inspection_tool class="MissingJavadoc" enabled="true" level="WARNING" enabled_by_default="true" />
-  <inspection_tool class="MissingOverrideAnnotation" enabled="true" level="WARNING" enabled_by_default="true" editorAttributes="WARNING_ATTRIBUTES"/>
-</profile>
\ No newline at end of file
+  <option name="myName" value="Default"/>
```

Comment:
shouldnt we introduce the XML merge namespace and root config?

Reply:
I've added the merge strategy and ids now.